hs_access_control/management/commands/community.py

```diff
@@ -0,0 +1,322 @@
+"""
+This allows creation of a community of groups without a graphical user interface.
+
+WARNING: As these routines run in administrative mode, no access control is used.
+Care must be taken to generate reasonable metadata, specifically, concerning
+who owns what. Non-sensical options are possible to create.
+This code is not a design pattern for actually interacting with communities.
+"""
+
+from django.core.management.base import BaseCommand
+from hs_access_control.models.community import Community
+from hs_access_control.models.privilege import PrivilegeCodes, \
+        UserGroupPrivilege, UserCommunityPrivilege, GroupCommunityPrivilege
+from hs_access_control.management.utilities import community_from_name_or_id, \
+        group_from_name_or_id, user_from_name
+
+
+def usage():
+    print("Community usage:")
+    print("  community [{cname} [{request} [{options}]]]")
+    print("Where:")
+    print("  {cname} is a community name. Use '' to embed spaces.")
+    print("  {request} is one of:")
+    print("      list: print the configuration of a community.")
+    print("      create: create the community.")
+    print("      update: update metadata for community.")
+    print("      Options for create and update include:")
```

Comment:
isn't this all provided with --help because you're using parser.add_argument

Reply:
It gives a prettier message than --help.